src/msha/handlers/function.handler.ts

```diff
@@ -53,8 +54,29 @@ function injectClientPrincipalCookies(req: http.ClientRequest) {
   }
 }
 
+async function resolveLocalhostByFetching(url: string) {
+  const { protocol, port } = parseUrl(url);
+  const fetchOneOfUrl = [`${protocol}//127.0.0.1:${port}`, `${protocol}//[::1]:${port}`];
+
+  for (let Url of fetchOneOfUrl) {
```

Comment:
Again, be careful with using `await` in `for` loops as it breaks asynchronicity.
You can use the same pattern as above to keep it asynchronous.

Reply:
Done. Thanks!